src/clusterfuzz/_internal/bot/tasks/trials.py

```diff
@@ -67,15 +68,16 @@ def __init__(self):
           continue
         self.trials[config['app_args']] = config['probability']
     except Exception:
-      logs.log_warn('Unable to parse config file because of the following error:')
+      logs.log_warn("""
+      Unable to parse config file because of the following error:
+      """)
       logs.log_warn(Exception)
```

Comment:
Not sure what that will do. The pattern for using a caught exception is something like:

except Exception as e:
logs.log_warn(str(e))
Or just add it to the log line above with %s. This isn't a standard logging method and it doesn't accept *args though.

Reply:
done